draft-ietf-httpbis-message-signatures.md

```diff
@@ -196,32 +199,78 @@ The following table shows example canonicalized values for header fields, given
 
 |Header Field|Canonicalized Value|
 |--- |--- |
-|`(cache-control)`|max-age=60, must-revalidate|
-|`(date)`|Tue, 07 Jun 2014 20:51:35 GMT|
-|`(server)`|www.example.com|
-|`(x-empty-header)`||
-|`(x-obs-fold-header)`|Obsolete line folding.|
-|`(x-ows-header)`|Leading and trailing whitespace.|
+|`cache-control`|max-age=60, must-revalidate|
+|`date`|Tue, 07 Jun 2014 20:51:35 GMT|
+|`server`|www.example.com|
+|`x-empty-header`||
+|`x-obs-fold-header`|Obsolete line folding.|
+|`x-ows-header`|Leading and trailing whitespace.|
 {: title="Non-normative examples of header field canonicalization."}
 
+## Dictionary Structured Field Members
+
+An individual member in the value of a Dictionary Structured Field is identified by the lowercased field name, followed by a semicolon `":"`, followed by the member name. An individual member in the value of a Dictionary Structured Field is canonicalized by applying the serialization algorithm described in Section 4.1.2 of {{StructuredFields}} on a Dictionary containing only that member.
```

Comment:
```suggestion
An individual member in the value of a Dictionary Structured Field is identified concatenating
the lowercased field name,
a semicolon `":"`
and the member name.
An individual member in the value of a Dictionary Structured Field is canonicalized by applying the serialization algorithm described in Section 4.1.2 of {{StructuredFields}} on a Dictionary containing only that member.
```

Reply:
@richanna following the discussion at the httpwg meeting I suggest to split this part in another PR so that we can land this one.

Sub-referencing header could be leveraged to push the reicever to parse a given header in a certain way which is in control of the sender (eg. triggering keyerror, indexerror, ..).

Reply:
There is also a typo - semicolon/colon